website/static/js/accountSettings.js

```diff
@@ -227,6 +227,66 @@ var UserProfileViewModel = oop.extend(ChangeMessageMixin, {
     }
 });
 
+
+var DeactivateAccountViewModel = oop.defclass({
+    constructor: function () {
+        this.success = ko.observable(false);
+    },
+    urls: {
+        'update': '/api/v1/profile/deactivate/'
+    },
+    submit: function () {
+        var request = $osf.postJSON(this.urls.update, {});
+        request.done(function() {
+            $osf.growl('Success', 'Deactivation request submitted.', 'success');
+            this.success(true);
+        }.bind(this));
+        request.fail(function(xhr, status, error) {
+            $osf.growl('Error',
+                'Deactivation request failed. Please contact <a href="mailto: support@cos.io">support@cos.io</a> if the problem persists.',
+                'danger'
+            );
+            Raven.captureMessage('Error requesting account deactivation', {
+                url: this.urls.update,
+                status: status,
+                error: error
+            });
+        }.bind(this));
+        return request;
+    }
+});
+
+
+var ExportAccountViewModel = oop.defclass({
+    constructor: function () {
+        this.success = ko.observable(false);
+    },
+    urls: {
+        'update': '/api/v1/profile/export/'
+    },
+    submit: function () {
+        var request = $osf.postJSON(this.urls.update, {});
+        request.done(function() {
+            $osf.growl('Success', 'Export request submitted.', 'success');
```

Comment:
We should probably let the user know what happens next, i.e. that we will review the request and get back to the user after some time. ping @lbanner @GaryKriebel for language